Dockerfile

```diff
@@ -39,55 +101,27 @@ COPY --from=lab-base /app/.env /lab/.env
 
 RUN sed -i "s/HOTJAR_IS_ENABLED=false/HOTJAR_IS_ENABLED=true/" /lab/.env
 
-# Building Mage and Memgraph
-RUN apt-get clean && \
-  apt-get update && \
-  apt-get install -f -y \
-  python3-setuptools \
-  build-essential \
-  cmake           \
-  curl            \
-  g++             \
-  git             \
-  netcat          \
-  libcurl4        \
-  libpython3.7    \
-  libssl1.1       \
-  openssl         \
-  python3         \
-  python3-pip     \
-  python3-dev     \
-  supervisor      \
-  --no-install-recommends
-
-RUN curl -fsSL https://deb.nodesource.com/setup_16.x | bash - \
-  && apt-get install -y nodejs \
-  && rm -rf /var/lib/apt/lists/* /tmp/* /var/tmp/*
-
-RUN pip3 install networkx==2.4 numpy==1.21.4 scipy==1.7.3
 
-ARG TARGETARCH
+ARG PY_VERSION_DEFAULT
+ENV PY_VERSION ${PY_VERSION_DEFAULT}
```

Comment:
These two are also repeated

Reply:
What do you mean? As I am aware, you need to reference ARG to use it in ENV. This is also due to fact that multi stage doesn't preserve ENV or ARG: more about it [here](https://github.com/moby/moby/issues/37345#issuecomment-400245466)

Reply:
These two lines are also on 8th and 9th line

Reply:
Fixed